homeassistant/components/totalconnect/__init__.py

```diff
@@ -1,17 +1,23 @@
 """The totalconnect component."""
+import asyncio
 import logging
 
 from total_connect_client import TotalConnectClient
-import voluptuous as vol
 
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
-from homeassistant.helpers import discovery
-import homeassistant.helpers.config_validation as cv
+from homeassistant.core import HomeAssistant
 
-_LOGGER = logging.getLogger(__name__)
+from .const import DOMAIN
+
+# import voluptuous as vol
+
+# import homeassistant.helpers.config_validation as cv
 
-DOMAIN = "totalconnect"
 
+_LOGGER = logging.getLogger(__name__)
+
+"""
```

Comment:
I don't think the config schema should be a string.

Reply:
removed